webapp/src/main/resources/public/js/stores/branches/BranchesHistoryStore.js

```diff
@@ -119,6 +135,12 @@ class BranchesHistoryStore {
         BranchesSearchParamsActions.changeEmpty(empty === "true");
         BranchesSearchParamsActions.changeNotEmpty(notEmpty === "true");
         BranchesSearchParamsActions.changeOnlyMyBranches(onlyMyBranches === "true");
+        if (createdBefore !== null) {
+            BranchesSearchParamsActions.changeCreatedBefore(moment(createdBefore))
```

Comment:
looks like that's where we want to apply those conversion. similar to what we do for `selectedBranchTextUnitIds`

Reply:
Handling as strings now, don't even need to convert!